conda/recipes/rapids-metadata/meta.yaml

```diff
@@ -0,0 +1,53 @@
+# Copyright (c) 2024, NVIDIA CORPORATION.
+
+{% set pyproject_data = load_file_data("pyproject.toml") %}
+{% set version = pyproject_data.get('project', {}).get('version') %}
```

Comment:
@jameslamb just started a conversation about moving away from `.get(...)`: https://github.com/rapidsai/build-planning/issues/72

Reply:
I've moved away from `.get(...)` and also removed the extra dependencies and the normal dependencies (since this project currently doesn't have any runtime dependencies).